packages/target-decisioning-engine/src/postProcessors.js

```diff
@@ -255,15 +255,13 @@ export function createResponseTokensPostProcessor(
       return accumulator;
     }, {});
 
-    const options = mboxResponse.options.map(option => {
-      return {
-        ...option,
-        responseTokens: {
-          ...responseTokensFromMeta,
-          ...responseTokens
-        }
-      };
-    });
+    const options = mboxResponse.options.map(option => ({
```

Comment:
it's a matter of opinion, but i liked it better before

Reply:
I don't have a strong opinion on this, however it was reported as an issue after ESLint upgrade and was fixed this way automatically after running `eslint --fix`